go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/openshift/client-go v0.0.0-20211209144617-7385dd6338e3
 	github.com/openshift/installer v0.9.0-master.0.20191219195746-103098955ced
 	github.com/openshift/library-go v0.0.0-20220110124235-d70e915b42bd
+	github.com/prometheus/client_golang v1.11.0
```

Comment:
Why not the latest? :)

Reply:
I didn't specifically choose the version, but my guess would be that the latest is incompatible with the rest of our packages. When I tried to `go get` the latest version now, it also updated a ton of other packages, so I'm not sure if that's something we want necessarily.

Reply:
Incompatible? In what sense (any example pls)? Interesting....it seems we will have to update it one day anyway.

Reply:
I don't know why it wants to update the other packages as well in this case, but generally speaking, it could simply be using a function/constant/whatever that isn't present in the version we're currently using, so it needs to update the dependency to a version that include that feature. It's not updating anything related to K8s, just some Prometheus side-libraries, Golang libraries that aren't built-in and `xxhash`.